src/pages/landing_page/LandingPage.tsx

```diff
@@ -0,0 +1,102 @@
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useEffect, useState } from 'react'
+
+import AuctionService from 'services/AuctionService'
+import GridLayout from 'shared/grid_layout/GridLayout'
+
+import './LandingPage.scss'
+
+const LandingPage = () => {
+
+	const [newArrivalsActive, setNewArrivalsActive] = useState(true)
+	const [lastChanceActive, setLastChanceActive] = useState(false)
+	const [auctions, setAuctions] = useState([])
+
+	useEffect(() => {
+		handleNewArrivals()
+	}, [])
+
+	const handleNewArrivals = () => {
+		setNewArrivalsActive(true)
+		setLastChanceActive(false)
+
+		AuctionService.getNewArrivals()
+			.then(response => {
+				if (response) {
+					setAuctions(response)
+				}
+			})
+	}
+
+	const handleLastChance = () => {
+		setNewArrivalsActive(false)
+		setLastChanceActive(true)
+
+		AuctionService.getLastChance()
+			.then(response => {
+				if (response) {
+					setAuctions(response)
+				}
+			})
+	}
+
+	let images = [
+		'https://media1.popsugar-assets.com/files/thumbor/CHzF5iQ31LcGCjSPu1xF0wjTypg/0x0:1500x2024/fit-in/1024x1024/filters:format_auto-!!-:strip_icc-!!-/2021/04/20/773/n/1922564/c9ce4a74607f107ac3b225.06048116_/i/Best-Women-Sneakers.jpg'
+	]
+
+	return (
+		<>
+			<div className="heading">
+				<div className="container">
+					<div className="row highlight">
+						<div className="col-12 col-sm-4 col-lg">
+							<h6 className="cat-title">CATEGORIES</h6>
+							<ul className="cat-list">
+								<li><div className="category"><a>Fashion</a></div></li>
+								<li><div className="category"><a>Accesories</a></div></li>
+								<li><div className="category"><a>Jewelry</a></div></li>
+								<li><div className="category"><a>Shoes</a></div></li>
+								<li><div className="category"><a>Sportware</a></div></li>
+								<li><div className="category"><a>Home</a></div></li>
+								<li><div className="category"><a>Electronics</a></div></li>
+								<li><div className="category"><a>Mobile</a></div></li>
+								<li><div className="category"><a>Computer</a></div></li>
+								<li><div className="category"><a>All Categories</a></div></li>
+							</ul>
+						</div>
+						<div className="col-12 col-sm-4 col-lg product-desc">
+							{auctions[0] ?
+								<>
+									<h4 className="prod-title">{auctions !== null ? auctions[0]['item']['name'] : ''}</h4>
+									<h4 className="price">Start from ${auctions[0]['item']['startPrice'] != null ? auctions[0]['item']['startPrice'] : ''}</h4>
+									<p>{auctions[0]['item']['description'] != null ? auctions[0]['item']['description'] : ''}</p> </> : ''
+							}
```

Comment:
I'm really struggling to read this. If you have some complex conditions or objects you need to form, you can do so outside of the direct return statement. By that, I mean that you can make a variable, conditionally assign the necessary value to it and just use `{ variable }` to display it here. That way you can make the logic more readable and it will still stay in one spot so you can change it in the future if necessary.